### PR TITLE
Update with all Contracts

### PR DESCRIPTION
**Pull request by ghickman**
This adds the changes from opensafely-core/databuilder#397 so we're including the Universal/Patient contract and removing the base TableContract.

### CODE REVIEW

**Review by iaindillingham — APPROVED** [review]
Ah, I see it now!

**Comment by ghickman** [comment]
I'll merge this after opensafely-core/databuilder#397 is in so we don't have docs from an unreleased version of databuilder.

File: docs/contracts-reference.md
Comment (ghickman) [reply docs/contracts-reference.md:14]: We have to manually add these in, which is how I missed it when first raising the PR.  Both backends and contracts work this way to give us control over what's rendered.  However, given that even I missed this off, maybe we should move to rendering all by default?

Comment (benbc) [reply docs/contracts-reference.md:14]: Yes, I think so. I don't think there's an obvious reason for having an undocumented contract. And if we discover one then we can add some kind of filter/exclusion thing later.

**Comment by ghickman** [comment]
@benbc – hierarchy added to the code, do you want a quick look before I merge?

**Comment by benbc** [comment]
@ghickman Do you mean the PR in the other repo? It's hard to have an opinion on that without knowing how you intend to use it, but I'm sure you know what you're doing. :-)

File: docs/contracts-reference.md
Comment (ghickman) [reply docs/contracts-reference.md:14]: I'll need to do this in a new PR as it requires changes to the plugin.

**Comment by ghickman** [comment]
@benbc – yes 🤦‍♂️

**Pull request merged**
